client/src/pages/DetailsPage.js

```diff
@@ -26,11 +28,31 @@ const Title = styled.h3`
 export const DetailsPage = () => {
   const { projectId } = useParams();
   const { user } = useUserState();
+  const [favoriteURL, setFavoriteURL] = useState("");
+  const [favoriteID, setFavoriteID] = useState("");
+  const [favoriteTitle, setFavoriteTitle] = useState("");
```

Comment:
I am not familiar with your project yet, but wouldn't it be easier to compile those states into one? (object)

Reply:
it would probably be easier, i'll have a look at that again